app/tests/integration/test_timeline_and_population.py

```diff
@@ -78,13 +78,3 @@ def test_firefox_max_version_updates_correctly(selenium, base_url, fill_overview
     version = "78.0"
     timeline_pop_form.firefox_max_version = version
     assert timeline_pop_form.firefox_max_version == version
-
-
-@pytest.mark.nondestructive
-def test_platform_selection_updates_correctly(selenium, base_url, fill_overview):
-    """Test platform selection updates."""
-    timeline_pop_form = fill_overview.save_and_continue_btn()
-    assert timeline_pop_form.platform == "All Platforms"
-    channel = "All Linux"
-    timeline_pop_form.platform = channel
-    assert timeline_pop_form.platform == channel
```

Comment:
@jrbenny35 showed me a better way to do this, just mark it as

`@pytest.mark.skip`

like so:

https://github.com/mozilla/experimenter/blob/f563df6936c06c1cf68096a57905e1b6d2e5a0f7/app/tests/integration/test_overview.py#L30

Reply:
ok so you want me to leave the test but just skip it?

Reply:
Yep exactly and then we can at least refer to it later for reference.